src/recipe/recipe.controller.ts

```diff
@@ -26,23 +28,41 @@ import {
 import { RecipeResponse } from './responses/recipe.response';
 import { User } from '@prisma/client';
 import { FindAllRecipesDTO } from './DTOs/find-all-recipes-query';
+import { FileInterceptor } from '@nestjs/platform-express';
 
 @Controller('/recipe')
 export class RecipeController {
   constructor(private readonly recipeService: RecipeService) {}
 
   @Post()
+  @UseInterceptors(FileInterceptor('file'))
   @UseGuards(AuthGuard('jwt'))
   @ApiOperation({ summary: 'Create a new recipe' })
   @ApiCreatedResponse({ type: RecipeResponse })
   @ApiUnauthorizedResponse({ description: 'Not logged in' })
   async createRecipe(
     @Body() recipe: CreateRecipeDTO,
+    @UploadedFile() file: Express.Multer.File,
```

Comment:
Why do you need this on createRecipe when you have a different endpoint to upload an image?

Reply:
i don't need it, my mistake, i removed this line

Reply:
Still not removed